src/music_gui.cpp

```diff
@@ -312,12 +314,30 @@ struct MusicTrackSelectionWindow : public Window {
 		this->LowerWidget(WID_MTS_ALL + _settings_client.music.playlist);
 	}
 
+	static DropDownList *BuildMusicSetsDropdown(int &selected_index)
+	{
+		DropDownList *musicsets = new DropDownList();
```

Comment:
Please ensure that "selected_index" gets initialised, without relying on BaseMusic::GetUsedSet not returning NULL or something